schemas/hash.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: hash
+  title: Group
+  group: 2
+  type: group
+  short: Hashes, usually file hashes.
+  description: >
+    The hash fields represent different hash algorithms and their values.
+
+    Field names for common hashes (e.g. MD5, SHA1) are predefined. Add fields for other hashes
+    by lowercasing the hash algorithm and using underscore separators as appropriate (snake case).
```

Comment:
Small clarification: "the hash algorithm name". I'd also give an example at the end of the sentence, something like "for example `sha3_512`"